RAAB7_scripts/RAAB7_reporter.Rmd

```diff
@@ -34,16 +34,14 @@ dir.create(here("outputs",ID,"/raw/data"))
 
 raab_merged<-read.csv(here("data", "surveys.csv"))
 raab<-raab_merged[raab_merged$raab_id==ID,]
-raab$finished<-as.Date(raab$finished)
+raab$date<-as.Date(raab$date)
 
 pop_merged<-read.csv(here("data", "population.csv"))
 pop<-pop_merged[pop_merged$raab_id==ID,]
 popfives<-pop
 
-mrm_vars<-c("raab_id","survey_title","year_end","iso_2","level1","level2","osm_id","iso3166-1","iso3166-2","admin_level","national","sample_size","response_rate","raab_version","va_threshold","raab_dr","raab_wgq","iceh_data","pi_name","pi_institution","pi_email1","trainer_name","trainer_email1","implementing_org","facilitating_org","funder","pub_doi1","pub_doi2","pub_doi3")
-
-meta<-read.csv(here("data","raab-log_v5.csv"))
-raab_meta<-meta[meta$raab_id==ID,mrm_vars]
+meta<-read.csv(here("data","meta.csv"))
```

Comment:
I could not get this to work, even setting the same cols in mrm_vars as in the csv:
```
"raab_id","raab_version","survey_title","year_end","iso_2","level1","level2","osm_id","iso3166-1","iso3166-2","admin_level","national","pi_name","pi_institution","pi_email1","trainer_name","trainer_email1","implementing_org","facilitating_org","funder","pub_doi1","pub_doi2","pub_doi3","data_access","embargo_date","sample_size","response_rate","raab_dr","raab_wgq"
```
I kept getting:
```
Quitting from lines 8-56 (RAAB7_reporter.Rmd)
Error in `[.data.frame`(meta, meta$raab_id == ID, mrm_vars) :
  undefined columns selected
```
removing the var gets the script to go ahead. From my understanding it's including all cols which should be the same thing.